charts/smokey/templates/cronjob.yaml

```
@@ -81,17 +82,21 @@ spec:
                   key: password
                   name: smokey-signon-account
             - name: SENTRY_CURRENT_ENV
-              value: {{ .Values.govukEnvironment }}
+              value: {{ $.Values.govukEnvironment }}
             - name: SENTRY_DSN
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.repoName }}-sentry
+                  name: {{ $.Values.repoName }}-sentry
                   key: dsn
             - name: SENTRY_RELEASE
-              value: "{{ .Values.appImage.tag }}"
+              value: "{{ $.Values.appImage.tag }}"
             - name: SE_CACHE_PATH
               value: "/tmp/.cache/selenium"
             - name: XDG_CONFIG_HOME
               value: "/tmp/.chrome"
             - name: XDG_CACHE_HOME
               value: "/tmp/.chrome"
+            {{- with .extraEnv }}
+              {{- (tpl (toYaml .) $) | trim | nindent 12 }}
```

Best to avoid the `tpl` hack until you definitely need it; otherwise template parsing tends to get slow (and it encourages overabstraction).
```suggestion
              {{- . | toYaml | nindent 12 }}
```